proxy/nginx.conf

```diff
@@ -29,6 +29,10 @@ http {
     listen {{port}};
     root public;
     index index.html index.htm Default.htm;
+
+    location = /robots.txt {
+        alias /public/robots.txt;
```

Comment:
@btylerburton, I feel like you've been in the `nginx.conf` files more recently than I have, does this seem like the right place to drop this?

Reply:
It is within (what I think is?) the first hit server block.

`nginx-common.conf` appears to be hit after `auth` stuff is, I think. I don't think we want `robots.txt` or the sitemaps to be behind the auth, right? I might not be understanding how auth is working in this case, though.

Reply:
check out this block then: https://github.com/GSA/catalog.data.gov/blob/main/proxy/nginx-common.conf#L19

we wanted health check not to be backed by auth and this is working

Reply:
I dropped it down by the `500` block. Thanks for this!